WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardPersonalizationService.swift

```diff
@@ -58,6 +58,8 @@ private func makeKey(for card: DashboardCard) -> String? {
         return "free-to-paid-plans-dashboard-card-enabled-site-settings"
     case .domainRegistration:
         return "register-domain-dashboard-card"
+    case .googleDomains:
+        return "domain-focus-card"
```

Comment:
```suggestion
        return "google-domains-enabled-site-settings"
```
This is to keep aligned with the pattern used by other cards and to avoid clashes in the future with other keys as the original key was a bit vague.